From d122792e03cd492238c41923a1f1c6774fc08312 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 28 Sep 2023 13:20:48 +0000
Subject: libstdc++: std::vector should not have const member in autofill_table

Bug: 957519
Change-Id: Ie31df26299bee166a4dc1ba33008422ed0d2fc90
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4892772
Reviewed-by: Stephen McGruer <smcgruer@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Auto-Submit: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1202565}
---
 components/autofill/core/browser/webdata/autofill_table.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/components/autofill/core/browser/webdata/autofill_table.cc b/components/autofill/core/browser/webdata/autofill_table.cc
index 28e21361b4883..d4446456d91f6 100644
--- a/components/autofill/core/browser/webdata/autofill_table.cc
+++ b/components/autofill/core/browser/webdata/autofill_table.cc
@@ -1713,7 +1713,7 @@ std::unique_ptr<AutofillProfile> AutofillTable::GetAutofillProfile(
     // VerificationStatus of the data entry's `value`.
     int status;
     // Serialized observations for the stored type.
-    std::vector<const uint8_t> serialized_data;
+    std::vector<uint8_t> serialized_data;
   };
 
   std::vector<FieldTypeData> field_type_values;
@@ -1735,8 +1735,8 @@ std::unique_ptr<AutofillProfile> AutofillTable::GetAutofillProfile(
     base::span<const uint8_t> observations_data = s.ColumnBlob(3);
     field_type_values.emplace_back(
         type, s.ColumnString16(1), s.ColumnInt(2),
-        std::vector<const uint8_t>(observations_data.begin(),
-                                   observations_data.end()));
+        std::vector<uint8_t>(observations_data.begin(),
+                             observations_data.end()));
 
     if (type == ADDRESS_HOME_COUNTRY) {
       country_code = base::UTF16ToUTF8(s.ColumnString16(1));
-- 
2.39.2

